### DISCUSSION TITLE
How to turn off multi-language functionallity?

### DISCUSSION THREAD

**Discussion by JerrelZ** [vhessam/hessam-cms#90] [Q&A] [answered]
Hi,

First of all, great package!

In version release 9.1 I read that you can turn off the multi-language functionality. 

Anyone knows how to achieve this?

Thanks,
Jerrel.

**Comment by samberrry** [comment, marked as answer]
Hi @JerrelZ, 

thank you :) 

The 9.x versions are all multi-language support by default. You can just keep using one language without adding more languages and just using website URLs without locales (e.g: /en). 

But if you do not need to have multi language support completely, you can use the 8.x version. But what is the difference, the main difference is DB structure. If you see that you will never have multi-lang content on your website (at least in the neat future), it would be more easier to use the 8.x version, the internal structure is easier as well. This way you will save number of queries, as the 8.x versions has lower number of tables

I hope it could help you and clarify. 

thanks

**Comment by JerrelZ** [comment]
Hi Samberry,

Thanks for your quick response.

I tried using 9.x but I got some errors while disabling the language code from the slugs. It's probably my own fault then so I will try again.

About the 8.x version, If I use that, won't it be outdated after a while? You're still updating this package right?

Thanks.

**Comment by samberrry** [comment]
Hi @JerrelZ,

no worries, report errors about disabling lang code from slugs, if you see any. 

I try to keep both up to dated, I still use this version 8.x.  

thanks

**Comment by mindrootstech** [comment]
Hi ,Add a comment not working it shows 401 UNAUTHORIZED
